src/main/java/com/gluonhq/substrate/target/IosTargetConfiguration.java

```diff
@@ -88,12 +107,66 @@ public String getAdditionalSourceFileLocation() {
         return Arrays.asList(objectFile.toAbsolutePath().toString());
     }
 
+    @Override
+    public boolean link(ProcessPaths paths, ProjectConfiguration projectConfiguration) throws IOException, InterruptedException {
+        boolean result = super.link(paths, projectConfiguration);
+
+        if (result) {
+            createInfoPlist(paths, projectConfiguration);
+
+            if (! isSimulator()) {
+                CodeSigning codeSigning = new CodeSigning(paths, projectConfiguration);
+                if (! codeSigning.signApp()) {
+                    Logger.logSevere("Error signing the app");
```

Comment:
and then what? Is this fatal? In that case, fail-fast.

Reply:
ok